src/main/java/ladder/domain/ladder/Stairs.java

```diff
@@ -0,0 +1,54 @@
+package ladder.domain.ladder;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Stairs {
+
+    private static final int EXCEPT_FIRST_AND_LAST_PILLAR_COUNT = 2;
```

Comment:
```suggestion
    private static final int FIRST_AND_LAST_PILLAR_COUNT = 2;
```
첫번째, 마지막 필러 카운트를 제외한 카운트가 2라는 의미처럼도 보여서 애매한거 같아요